smart-lungs/src/components/question-form.tsx

```diff
@@ -23,25 +23,37 @@ import {
   SelectValue,
 } from "./ui/select";
 import { cn } from "@/lib/utils";
+import { useState } from "react";
 
 const formSchema = z.object({
+  // Name, location and allergies are not used in the bad model. This is to mimic the privacy issue of unwanted data collection. (Not actually implemented)
   name: z.string().min(2, { message: "Name must be at least 2 characters." }),
-  location: z.string().min(2, { message: "Location must be at least 2 characters." }),
+  location: z
+    .string()
+    .min(2, { message: "Location must be at least 2 characters." }),
   age: z
     .number()
     .int()
     .min(16, { message: "You must be at least 16 years old." }),
-  gender: z.enum(["male", "female"]),
-  smoking: z.enum(["never", "rarely", "often", "daily"]),
-  allergies: z.string().min(2, { message: "Allergies must be at least 2 characters long." }),
+  gender: z.enum(["0", "1"]),
+  smoking: z.enum(["0", "1", "2", "3"]),
+  shortnessOfBreath: z.enum(["0", "1", "2", "3"]),
+  peerPressure: z.enum(["0", "1", "2", "3"]),
+  fatigue: z.enum(["0", "1", "2", "3"]),
+  allergies: z
+    .string()
+    .min(2, { message: "Allergies must be at least 2 characters long." }),
 });
```

Comment:
I don't think we need allergies at all here, not added benefits to the bad app.

Reply:
Removed, you can merge the PR now if everything is in order.